### PR TITLE
chore: Adding a new Bug Verification page

### PR DESCRIPTION
**Pull request by ArakTaiRoth**

### CODE REVIEW

**Comment by lizhuomeng71** [comment]
Is the PR ready for test?

**Comment by lizhuomeng71** [comment]
Typo 
"design-systen-support" should be corrected to "design-system-support."

image

File: src/routes/get-started/developers/BugVerification.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<p>This process is for Angular or React users only.</p>
+  
+  <h3>1. Version check</h3>
+      <p>Ensure you are using the latest version of:</p>
+      <ul>
+      <li>Web components</li>
+      <li> Angular or React</li>
+      </ul>
+
+      <p>If not, upgrade and retest. If yes proceed to step 2.</p>
```
Comment (Spark450) [inline src/routes/get-started/developers/BugVerification.tsx:19]: Add comma after "yes" on this line

File: src/routes/get-started/developers/BugVerification.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <li>Verify if the issue can be replicated</li>
+</ul>
+      <h3>4. Sharing and reporting</h3>
+      <p>If the issue is replicable:</p>
+      <ul>
+      <li>Share the template app link in the <a href="https://goa-dio.slack.com/archives/C02PLLT9HQ9">#design-systen-support</a> channel</li>
+        <li>Provide a link to your repo where we can quickly view and replicate the issue</li>
+      </ul>
+
+      <p>By following these steps, you'll help ensure efficient investigation and quicker resolution of design system component issues.</p>
```
Comment (Spark450) [inline src/routes/get-started/developers/BugVerification.tsx:43]: replace this line with "Following these steps will help to ensure efficient investigation and quicker resolution of design system component issues."

**Review by Spark450 — CHANGES_REQUESTED** [review]
Just 2 small requested changes, looks good!

**Comment by asikinack** [comment]
Completed both content changes suggest by Mark.

File: src/routes/get-started/developers/BugVerification.tsx (hunk 2 above)
Comment (asikinack) [reply src/routes/get-started/developers/BugVerification.tsx]: Done

File: src/routes/get-started/developers/BugVerification.tsx (hunk 1 above)
Comment (asikinack) [reply src/routes/get-started/developers/BugVerification.tsx]: Done.

**Review by Spark450 — APPROVED** [review]
Changes look good!

**Pull request merged**